src/amber/cli/templates/app/config/database.yml.ecr

```diff
@@ -3,7 +3,7 @@
    when "mysql" -%>
   database: mysql://root@localhost:3306/<%= "#{@name}_development" %>
 <% when "pg" -%>
-  database: postgres://postgres:@pg:5432/<%= "#{@name}_development" %>
+  database: postgres://root:@localhost:5432/<%= "#{@name}_development" %>
```

Comment:
🎉

Reply:
Yes, I opened a ticket https://github.com/amberframework/granite-orm/issues/67 to look at supporting Amber new environment yml.  Not this PR though.